tutorials/misc/plot_seeg.py

```diff
@@ -0,0 +1,149 @@
+
+###############################################################################
+# Now we make a :class:`mne.channels.DigMontage` stating that the sEEG
+# contacts are in the FreeSurfer surface RAS (i.e., MRI) coordinate system.
+
+coord_frame = 'mri'
+montage = mne.channels.make_dig_montage(
+    ch_pos, coord_frame=coord_frame, nasion=nasion, lpa=lpa, rpa=rpa)
+print('Created %s channel positions' % len(ch_names))
+
+###############################################################################
+# Now we get the :term:`trans` that transforms from our MRI coordinate system
+# to the head coordinate frame. This transform will be applied to the
+# data when applying the montage so that standard plotting functions like
+# :func:`mne.viz.plot_evoked_topomap` will be aligned properly.
+
+trans = mne.channels.compute_native_head_t(montage)
+print(trans)
+
+###############################################################################
+# Now that we have our montage, we can load in our corresponding
+# time-series data and set the montage to the raw data.
+
+# first we'll load in the sample dataset
+raw = mne.io.read_raw_edf(misc_path + '/seeg/sample_seeg.edf')
+
+# drop bad channels
+raw.info['bads'].extend([ch for ch in raw.ch_names if ch not in ch_names])
+raw.load_data()
+raw.drop_channels(raw.info['bads'])
+raw.crop(0, 2)  # just process 2 sec of data for speed
+
+# attach montage
+raw.set_montage(montage)
+
+# set channel types to sEEG (instead of EEG)
+raw.set_channel_types({ch_name: 'seeg' for ch_name in raw.ch_names})
+
+###############################################################################
+# We can then plot the locations of our electrodes on our subject's brain.
+# We'll use :func:`~mne.viz.snapshot_brain_montage` to save the plot as image
+# data (along with xy positions of each electrode in the image), so that later
+# we can plot frequency band power on top of it.
+#
+# .. note:: These are not real electrodes for this subject, so they
+#           do not align to the cortical surface perfectly.
+
+# fig = plot_alignment(raw.info, subject=subject, subjects_dir=subjects_dir,
+#                      surfaces=['pial'], trans=trans, coord_frame=coord_frame)
+# mne.viz.set_3d_view(fig, 200, 70, focalpoint=[0, -0.005, 0.03])
+#
+# xy, im = snapshot_brain_montage(fig, montage)
+
+
+###############################################################################
+# Next, we'll compute the signal power in the gamma (30-90 Hz) and alpha
+# (8-12 Hz) bands.
+gamma_power_t = raw.copy().filter(30, 90).apply_hilbert(
+    envelope=True).get_data()
+alpha_power_t = raw.copy().filter(8, 12).apply_hilbert(
+    envelope=True).get_data()
+gamma_power = gamma_power_t.mean(axis=-1)
+alpha_power = alpha_power_t.mean(axis=-1)
+
+# choose a colormap range wide enough for both frequency bands
+_gamma_alpha_power = np.concatenate((gamma_power, alpha_power)).flatten()
+vmin, vmax = np.percentile(_gamma_alpha_power, [10, 90])
```

Comment:
Maybe we should just plot raw traces in this example?

Reply:
Okay took this part out, and just added a `raw.plot()` to show that plotting SEEG data raw.